api/src/core/sql/schema/final.schema.sql

```diff
@@ -247,6 +250,10 @@ create table words(
   created_by bigint not null references users(user_id),
   unique nulls not distinct (wordlike_string_id, language_code, dialect_code, geo_code)
 );
+create index idx_wordlike_string_id on words (wordlike_string_id);
+create index idx_language_code on words (language_code);
```

Comment:
When we start searching using `dialect` and `geo`, we will always search in order of `language`, `dialect`, `geo`. Because of this we should just create:
`create index idx_language_codes on words (language_code, dialect_code, geo_code);`
and remove the separate indexes for `dialect` and `geo`